src/frontend/src/routes/router.tsx

```diff
@@ -37,6 +37,10 @@ const Router = () => {
         <Route path="/pia-intake" element={<PIAIntakeFormPage />} />
         <Route path="/pia-result" element={<PIAIntakeResultsPage />} />
         <Route path="/pia-list" element={<PIAList />} />
+        <Route
+          path="/pia/result/:id/:title"
+          element={<div> not implemented yet </div>}
```

Comment:
is the `element` required field here? I was hoping to get rid of it altogether if not implemented.